kotlin/internal/toolchains.bzl

```diff
@@ -107,6 +108,10 @@ _kt_toolchain = rule(
                 "1.3",
             ],
         ),
+        "extra_copts": attr.string_list(
```

Comment:
Should probably follow naming from `java_toolchain` and be `kotlincopts` https://docs.bazel.build/versions/master/be/java.html#java_toolchain.javacopts

Reply:
For context I plan to add `javacopts` to `kt_jvm_library` later, having `kotlinc` and `javac` prefixes is prob best to avoid confusion while also being on same page with `rules_java`

Reply:
Gonna lean towards a hard no on adding javacopts to the kt_jvm_lib. When IR becomes available, it would get dicey.

Reply:
just quick question: what is `IR`?

Reply:
>Gonna lean towards a hard no on adding javacopts to the kt_jvm_lib. When IR becomes available, it would get dicey.

Hold on, `javacoptions` are for `javac` call for `.java` sources in mixed Kotlin/Java `kt_jvm_library`, they have nothing to do with `kotlinc` options :)

>just quick question: what is `IR`?

Intermediate Representation, form of abstract (byte)code produced by Kotlin toolchain before it is transformed to actual instructions of the target platform

Reply:
Normally, I'd agree, but the kotlin java compilation is currently interleaved. I'm taking a wait and see approach.